R/pkgs/covidcommon/R/USAFacts.R

```diff
@@ -0,0 +1,71 @@
+##'
+##' Download USAFacts data
+##'
+##' Downloads the USAFacts case and death count data
+##'
+##' @param filename where case data will be stored
+##' @param url URL to CSV on USAFacts website
+##' @return data frame
+##'
+##' @importFrom dplyr select rename filter mutate
+##' @importFrom lubridate mdy
+##' @importFrom readr read_csv col_character
+##' @importFrom tidyr pivot_longer
+##'
```

Comment:
should this be exported? or are functions still not exported in covidcommon?

Reply:
This is more of an internal helper function. So I didn't export it. Just get_USAFacts_data()